src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,19 @@
+package duke.command;
```

Comment:
First of all, excellent job! I really like this abstraction of command as a class, it makes the main code much neater.